docs/getting-started.stories.mdx

```diff
@@ -37,6 +37,23 @@ There is a peer dependency on `draft-js` any project intending to use either the
 npm install draft-js@^0.11.5
 ```
 
+#### Global stylesheet
+We recommend using our global stylesheet which could be imported:
+
+```
+import GlobalStyle from 'carbon-react/lib/style/global-style';
+```
+
+then the `GlobalStyle` component could be placed next to the content of the app:
+```
+  <App>
+    <GlobalStyle />
+    .
+    .
+    .
+  </App>
+```
+
```

Comment:
I think we want to provide the global CSS in here as well, in case the user just wants to copy and paste it into their own global CSS rather than use our `GlobalStyle` component

Reply:
good point

Reply:
We have decided against this, we want to encourage the use of `GlobalStyle`